projects/pgai/pgai/vectorizer/embedders/__init__.py

```diff
@@ -1,5 +1,3 @@
-from .ollama import Ollama
-from .openai import OpenAI
-from .voyageai import VoyageAI
-
-__all__ = ["Ollama", "OpenAI", "VoyageAI"]
+from .ollama import Ollama as Ollama
```

Comment:
The redundant alias is needed, otherwise the linter will complain and provide the only alternative of adding it to the `__all__` var